cli/src/account.rs

```diff
@@ -55,154 +98,173 @@ pub enum AccountPromptResponse {
                 }
-                AccountCommand::MintNft {
-                    address,
-                    immutable_metadata_hex,
-                    immutable_metadata_file,
-                    metadata_hex,
-                    metadata_file,
-                    tag,
-                    sender,
-                    issuer,
-                } => {
-                    mint_nft_command(
-                        account,
-                        address,
-                        bytes_from_hex_or_file(immutable_metadata_hex, immutable_metadata_file).await?,
-                        bytes_from_hex_or_file(metadata_hex, metadata_file).await?,
-                        tag,
-                        sender,
-                        issuer,
-                    )
-                    .await
+                "accounts" => {
+                    // List all accounts
+                    let accounts = wallet.get_accounts().await?;
+                    println!("INDEX\tALIAS");
+                    for account in accounts {
+                        let details = &*account.details().await;
+                        println!("{}\t{}", details.index(), details.alias());
+                    }
                 }
-                AccountCommand::NewAddress => new_address_command(account).await,
-                AccountCommand::NodeInfo => node_info_command(account).await,
-                AccountCommand::Output { output_id } => output_command(account, output_id).await,
-                AccountCommand::Outputs => outputs_command(account).await,
-                AccountCommand::Send {
-                    address,
-                    amount,
-                    return_address,
-                    expiration,
-                    allow_micro_amount,
-                } => {
-                    let allow_micro_amount = if return_address.is_some() || expiration.is_some() {
-                        true
-                    } else {
-                        allow_micro_amount
+                _ => {
+                    // Prepend `Account: ` so the parsing will be correct
+                    let command = format!("Account: {}", command.trim());
+                    let account_cli = match AccountCli::try_parse_from(command.split_whitespace()) {
+                        Ok(account_cli) => account_cli,
+                        Err(err) => {
+                            println!("{err}");
+                            return Ok(AccountPromptResponse::Reprompt);
+                        }
                     };
-                    send_command(
-                        account,
-                        address,
-                        amount,
-                        return_address,
-                        expiration.map(|e| e.as_secs() as u32),
-                        allow_micro_amount,
-                    )
-                    .await
+                    if let Err(err) = match account_cli.command {
```

Comment:
I think we should move the error check to below and replace this with `let res = `

Reply:
I changed it, but didn't add the extra `res` binding ... just did an `unwrap_or_else` after the match block. Fine with you?

Reply:
sure